presto-raptor/src/test/java/com/facebook/presto/raptor/RaptorQueryRunner.java

```diff
@@ -70,6 +70,7 @@ public static DistributedQueryRunner createRaptorQueryRunner(
                 .put("metadata.db.filename", new File(baseDir, "db").getAbsolutePath())
                 .put("storage.data-directory", new File(baseDir, "data").getAbsolutePath())
                 .put("storage.max-shard-rows", "2000")
+                .put("storage.max-files-created", "200")
```

Comment:
We don't have to have this.

Reply:
This is for testing the error behavior in `TestRaptorSmokeIntegrationTests`. Also, it may make sense to limit the number of files created by unit tests as well.